portfolio/src/main/webapp/introduction.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lorraine Lyu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <div class="main">
+        <h1>A Brief Introduction about Me</h1>
+        <p>My name is <b>Lorraine Lyu</b>!</p>
+        <p>I'm a sophomore from <b>Rice University</b>.</p>
+        <p>My current major is <b>Computer Science</b> and 
+          I'm also interested in <b>Math&Studio Art& Psychology.</b></p>
+        <p>I draw/paint/do digital painting/do Chinese painting in my free time.</p>
+        <p>My favorite snack is <b>Wasabi flavored seaweed</b>.</p>
+        <p>I'm an actress in my school's theatre club.</p>
+        <p>I had my high school education in <b>Jiangsu, China.</b></p>
+        <p>For me, my high school is one of the most beautiful places in the world.</p>
+        <img src="images/suzhong1.jpg" alt="Image Of My High School">
```

Comment:
Same feedback as the other review. You can just write "My high school" in the alt text here and below.